CMakeLists.txt

```diff
@@ -244,7 +244,7 @@ if (MFEM_USE_SUNDIALS)
     find_package(SUNDIALS REQUIRED NVector_Serial CVODES ARKODE KINSOL)
   else()
     find_package(SUNDIALS REQUIRED
-      NVector_Serial NVector_Parallel NVector_ParHyp CVODES ARKODE KINSOL)
+      NVector_Serial NVector_Parallel NVector_MPIPlusX CVODES ARKODE KINSOL)
```

Comment:
What about the CUDA dependency in the CMake build system?

Reply:
I have added a link to handle this.